crates/admin-rest-model/src/version.rs

```diff
@@ -23,13 +23,46 @@ use std::ops::RangeInclusive;
 pub enum AdminApiVersion {
     Unknown = 0,
     V1 = 1,
+    V2 = 2,
+}
+
+impl From<AdminApiVersion> for (http::HeaderName, http::HeaderValue) {
+    fn from(value: AdminApiVersion) -> Self {
+        (
+            AdminApiVersion::HEADER_NAME,
+            http::HeaderValue::from(value.as_repr()),
```

Comment:
Did we come to a conclusion how to encode the different versions (header vs. path)?

Reply:
we have not